bigquery/README.md

```diff
@@ -2,50 +2,50 @@
 
 ### Quickstart
 
-Index a [default public BigQuery table](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.sample_info)
-into an Elasticsearch container on your local machine.
-
-* [Install bq](https://cloud.google.com/bigquery/docs/bq-command-line-tool#installation)
-if you haven't done so already.
-* Copy table to your project.
-  ```
-  bq --project_id MY_GOOGLE_CLOUD_PROJECT mk platinum_genomes
-  bq --project_id MY_GOOGLE_CLOUD_PROJECT cp google.com:biggene:platinum_genomes.sample_info  MY_GOOGLE_CLOUD_PROJECT:platinum_genomes.sample_info
-  ```
-* Change project ids in `dataset_config/platinum_genomes/facet_fields.csv`
-* If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`
-* `docker-compose up --build`
-* View Elasticsearch index:
- `http://localhost:9200/platinum_genomes/_search?pretty=true`
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
+* Determine the project that will be billed for querying the BigQuery tables.
+Your account must have `bigquery.jobs.create` permissions this project; this
+includes any project where you have the Viewer/Editor/Owner role.
+* Run the indexer:
```

Comment:
Change back to no "Run the indexer", this command runs indexer and Elasticsearch

Reply:
Done